lib/galaxy/schema/invocation.py

```diff
@@ -516,10 +563,28 @@ class WorkflowInvocationResponse(Model):
     output_values: Dict[str, Any] = Field(
         default=Required, title="Output values", description="Output values of the workflow invocation."
     )
-    messages: List[InvocationMessageResponseUnion] = Field(
-        default=Required,
-        title="Messages",
-        description="A list of messages about why the invocation did not succeed.",
+
+
+class WorkflowInvocationCollectionView(WorkflowInvocationBaseResponse):
+    steps: Optional[List[InvocationStep]] = Field(
+        default=None, title="Steps", description="Steps of the workflow invocation."
+    )
+    inputs: Optional[Dict[str, InvocationInput]] = Field(
+        default=None, title="Inputs", description="Input datasets/dataset collections of the workflow invocation."
+    )
+    input_step_parameters: Optional[Dict[str, InvocationInputParameter]] = Field(
+        default=None, title="Input step parameters", description="Input step parameters of the workflow invocation."
+    )
+    outputs: Optional[Dict[str, InvocationOutput]] = Field(
+        default=None, title="Outputs", description="Output datasets of the workflow invocation."
+    )
+    output_collections: Optional[Dict[str, InvocationOutputCollection]] = Field(
+        default=None,
+        title="Output collections",
+        description="Output dataset collections of the workflow invocation.",
+    )
+    output_values: Optional[Dict[str, Any]] = Field(
+        default=None, title="Output values", description="Output values of the workflow invocation."
```

Comment:
```suggestion
    pass
```
The collection view has none of these fields.